Scripts/pre_processing.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
maybe rename `pre_processing` to `preprocessing`

Reply:
Thanks I will rename it . I tested it on a folder with 500 files at a time but yeah we can handle if the problem occurs later.